R/symptome_compartment_class.R

```diff
@@ -0,0 +1,226 @@
+#' An SEIR model for compartments of different symptoms
+#'
+#' Note:
+#' 1) Sum of poulation equals to 1 at any time point.
```

Comment:
"population" -- I'd actually change this to be "Total population size is normalised to 1."